.pipelines/templates/steps/load-variables.yml

```diff
@@ -0,0 +1,45 @@
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+#
+# THIS CODE AND INFORMATION ARE PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED WARRANTIES 
+# OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+# ----------------------------------------------------------------------------------
+
+steps:
+
+# Load Variables
+- task: Bash@3
+  displayName: Load Variables
+  inputs:
+    targetType: inline
+    script: |
+      $(var-bashPreInjectScript)
+
+      ### -------------------------------------------------------------------------------------------------
+      ### Create the legacy 'var-parentManagementGroupId' and 'var-topLevelManagementGroupName'
```

Comment:
These two parameters are not legacy since they are still required for pipelines to function.  The automatic approach is to query the hierarchy and set there value.

Therefore, I wouldn't call them legacy.  We will keep these parameters indefinitely.

Reply:
Removed 'legacy' from this step (3 occurrences)